alpha/apps/kaltura/lib/storage/kStorageExporter.php

```diff
@@ -596,6 +616,25 @@ public static function handleAssetStorageExports($object)
 		}
 	}
 
+	protected static function isSourceFlavorAsset($asset)
+	{
+		if( ($asset->getType() == assetType::FLAVOR) && ($asset->getFlavorParamsId() == flavorParams::SOURCE_FLAVOR_ID) )
+		{
+			return true;
+		}
+		return false;
+	}
+
+	protected static function handleFileSyncStorageExports($object)
```

Comment:
if the argument for this function must be file sync than add the type to the signature to avoid future confusiom